test/e2e/common/node/sysctl.go

```diff
@@ -111,23 +112,17 @@ var _ = SIGDescribe("Sysctls [LinuxOnly] [NodeFeature:Sysctls]", func() {
 		gomega.Expect(log).To(gomega.ContainSubstring("kernel.shm_rmid_forced = 1"))
 	})
 
-	/*
-	  Release: v1.21
-	  Testname: Sysctl, allow specified unsafe sysctls
-	  Description: Pod is created with kernel.shm_rmid_forced. Should allow unsafe sysctls that are specified.
-	  [LinuxOnly]: This test is marked as LinuxOnly since Windows does not support sysctls
-	*/
-	framework.ConformanceIt("should support unsafe sysctls which are actually allowed [MinimumKubeletVersion:1.21]", func() {
+	ginkgo.It("should forbidden unsafe sysctls [MinimumKubeletVersion:1.21]", func() {
 		pod := testPod()
 		pod.Spec.SecurityContext = &v1.PodSecurityContext{
 			Sysctls: []v1.Sysctl{
 				{
-					Name:  "kernel.shm_rmid_forced",
-					Value: "1",
+					Name:  "net.core.somaxconn",
+					Value: "1024",
 				},
 			},
 		}
-		pod.Spec.Containers[0].Command = []string{"/bin/sysctl", "kernel.shm_rmid_forced"}
+		pod.Spec.Containers[0].Command = []string{"/bin/sysctl", "net.core.somaxconn"}
 
 		ginkgo.By("Creating a pod with the kernel.shm_rmid_forced sysctl")
```

Comment:
Don't we need to update this message instead of kernel.shm_rmid_forced sysctl?

Reply:
My careless. Thanks.

Updated.